README.md

```diff
@@ -48,6 +48,28 @@ shut down immediately. They leave no traces (outside of `/nix/store`) on the hos
   NixOps can be used to deploy to various other backends like cloud providers.\
   Requires: [Nix](https://nixos.org/nix/), [VirtualBox](https://www.virtualbox.org)
 
+#### Tests
+The internal test suite is also useful for exploring features.  
+The following `run-tests.sh` commands leave no traces (outside of `/nix/store`) on
+the host system.
+
+```bash
+git clone https://github.com/fort-nix/nix-bitcoin
+cd nix-bitcoin/test
+
+# Run a Python test shell inside a VM node
+./run-tests.sh debug
+print(succeed("systemctl status bitcoind"))
+
+# Run a node in a container. Requires systemd.
```

Comment:
... and root. At least this is the case for me on Arch and NixOS. Or am I doing something wrong?

Reply:
Yes, until NixOS adds user containers. Fixed via [force-push](https://github.com/fort-nix/nix-bitcoin/compare/474095a3951009ebdef1a1e9bd260ecc26cb04a8..d586b47dd425c2424dbbfaaf62bd719216d44f8e).

Reply:
FYI, here's the related [PR (WIP)](https://github.com/NixOS/nixpkgs/pull/67336).

Reply:
Thanks, subscribed